external/storm-solr/src/main/java/org/apache/storm/solr/bolt/SolrUpdateBolt.java

```diff
@@ -59,16 +63,29 @@ public SolrUpdateBolt(SolrConfig solrConfig, SolrMapper solrMapper, SolrCommitSt
         this.solrConfig = solrConfig;
         this.solrMapper = solrMapper;
         this.commitStgy = commitStgy;
-        logger.debug("Created {} with the following configuration: " +
+        LOG.debug("Created {} with the following configuration: " +
                     "[SolrConfig = {}], [SolrMapper = {}], [CommitStgy = {}]",
                     this.getClass().getSimpleName(), solrConfig, solrMapper, commitStgy);
     }
 
+    @Override
     public void prepare(Map stormConf, TopologyContext context, OutputCollector collector) {
         this.collector = collector;
         this.solrClient = new CloudSolrClient(solrConfig.getZkHostString());
         this.toCommitTuples = new ArrayList<>(capacity());
+        this.tickTupleInterval = solrConfig.getTickTupleInterval();
```

Comment:
@vesense I would suggest that all the code handling the tickTupleInterval goes inside the method setTickTupleInterval. It is cleaner then setting the field tickTupleInterval in prepare, and then have an if, and then set the field again in setTickTupleInterval. The default case should be handled there as well.